MonolithDev/gettingSongs/get_playlist_songs.py

```
@@ -0,0 +1,150 @@
+        elif "spotify.com/playlist/" in playlist_uri:
+            playlist_id = playlist_uri.split("/")[-1].split("?")[0]
+        else:
+            playlist_id = playlist_uri
+
+        # Get playlist tracks
+        raw_results = sp.playlist_tracks(playlist_id)
+
+        # Parse response using Pydantic model
+        try:
+            playlist_response = PlaylistResponse.model_validate(raw_results)
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)